core/src/validator.rs

```diff
@@ -2211,16 +2270,14 @@ fn blockstore_contains_incorrect_shred_version(
 
 /// If the blockstore contains any shreds with the incorrect shred version,
 /// copy them to a backup blockstore and purge them from the actual blockstore.
-fn backup_and_clear_blockstore(
-    ledger_path: &Path,
+fn check_blockstore_for_incorrect_shred_version(
```

Comment:
Decided to rename as the old name wasn't super descriptive IMO. Open to suggestions if you have anything better - I think it is nice that the names line up with the new function `should_check_blockstore_*` and `check_blockstore_*`

Reply:
Maybe something like `cleanup_blockstore_incorrect_shred_versions`? `check` implies that this is pure but we are modifying blockstore here.

Then the other one could be `should_cleanup_` or something similar.

Reply:
> check implies (to me) that this is pure but we are modifying blockstore here.

Same here, I like your suggestion so will update to that!

Reply:
https://github.com/anza-xyz/agave/pull/2851/commits/3519842e2b0d5d48eadf6829fe03bc5b9d9269cf